source/extensions/filters/network/dubbo_proxy/active_message.cc

```diff
@@ -15,8 +15,7 @@ ActiveResponseDecoder::ActiveResponseDecoder(ActiveMessage& parent, DubboFilterS
                                              ProtocolPtr&& protocol)
     : parent_(parent), stats_(stats), response_connection_(connection),
       protocol_(std::move(protocol)),
-      decoder_(std::make_unique<ResponseDecoder>(*protocol_, *this)), complete_(false),
-      response_status_(DubboFilters::UpstreamResponseStatus::MoreData) {}
+      decoder_(std::make_unique<ResponseDecoder>(*protocol_, *this)), complete_(false) {}
```

Comment:
Seems if we declare the member by type like `bool complete_ : 1;`, then clang tidy want complain the initializer list.

Reply:
lets land for now we can follow up and/or adjust following prs